packages/create-toolpad-app/src/package.ts

```diff
@@ -0,0 +1,27 @@
+import path from 'path';
+import * as fs from 'fs/promises';
+import type { PackageManager } from './types';
+
+export function getPackageManager(): PackageManager {
+  const userAgent = process.env.npm_config_user_agent;
+
+  if (userAgent) {
+    if (userAgent.startsWith('yarn')) {
+      return 'yarn';
+    }
+    if (userAgent.startsWith('pnpm')) {
+      return 'pnpm';
+    }
+    if (userAgent.startsWith('npm')) {
+      return 'npm';
+    }
+  }
+  return 'pnpm';
+}
+
+export async function findCtaPackageJson() {
```

Comment:
There exist package json types in this workspace already. Should add a return type here.